std/machines/arith16.asm

```diff
@@ -10,45 +10,43 @@ use std::convert::expr;
 use std::prover::eval;
 use std::prelude::Query;
 use std::machines::range::Byte;
+use std::machines::range::Byte2;
 
-// Arithmetic machine, ported mainly from Polygon: https://github.com/0xPolygonHermez/zkevm-proverjs/blob/main/pil/arith.pil
-// This machine supports eq0, which is the affine equation. Currently we only expose operations for mul and div.
-machine Arith16(byte: Byte) with
+// Arithmetic machine, inspired by Polygon's 256-Bit Arith machine: https://github.com/0xPolygonHermez/zkevm-proverjs/blob/main/pil/arith.pil
+machine Arith16(byte: Byte, byte2: Byte2) with
     latch: CLK8_7,
-    operation_id: operation_id,
+    operation_id: is_division,
     // Allow this machine to be connected via a permutation
     call_selectors: sel,
 {
-    col witness operation_id;
-
-    // operation_id has to be either mul or div.
-    force_bool(operation_id);
```

Comment:
This is ensured because there is no operation ID other than 0 or 1, so there won't be a lookup / permutation setting it to anything else.